### ISSUE TITLE
Add Item - File upload error message is not announced to screen readers

### ISSUE THREAD

**Issue by aviannamiller** [omeka/Omeka#1135]
"Description: On the ‘Add Item’ page, in the ‘Files’ tab there’s a ‘Choose Files’ button and right above that button is a visual instruction saying, ‘The maximum file size is 128 MB’. If users choose a file that exceeds 128 MB and they attempt to add the item with the ‘Add Item’ button, they are given a visual error message. Screen readers on the other hand are not being made aware of the error at all, and they do not announce the error message. 

Expected Results: Screen reader users should be able to identify if there is an error associated with the input given. 

Actual Results: The error message fails to be announced to screen reader users."

4.1.3 Status Messages; Level AA; WCAG 2.1
